kedro_projetaai/cli/projetaai_extras.py

```diff
@@ -0,0 +1,33 @@
+import click
+import os
+import shutil
+from kedro_projetaai.utils.datasets import extras
+from kedro.framework.context import KedroContext
+import toml
+
+
+@click.group()
+def projetaai_extras():
+    """custom projetaai cli"""
+    pass
+
+
+@projetaai_extras.command()
+def extra_datasets():
+    """
+    add custom datasets to kedro"""
+    click.echo("copying extras datasets to kedro catalog")
+    project_name = toml.load("pyproject.toml")["tool"]["kedro"]["project_name"]
```

Comment:
```suggestion
    project_name = kedro.framework.project.PACKAGE_NAME
```
[Referencia](https://github.com/kedro-org/kedro/blob/160fd6bc66d4d780652dd77499abeb23e13f1c09/kedro/framework/project/__init__.py#L230)

Reply:
po procurei muito isso e não achei kkkkkkkk